### DISCUSSION TITLE
Django 4.2 and Python 3.11 Support

### DISCUSSION THREAD

**Discussion by oniflix** [encode/django-rest-framework#8940] [Question & Answer] [answered]
I was eagerly waiting for the release of Django 4.2 so that I could communicate with a PostgreSQL database using the psycopg module. Later on, I realized that creating an API would be useful for my project. However, I found out that the current version of Django REST framework does not yet support Django 4.2 and Python 3.11. 
I would like to know if you have any information about when this support will be available. 
If not, do you have any suggestions for making internal modifications to make the framework functional with Django 4.2 and Python 3.11? Or should I revert to Python 3.10 and Django 4.1 with psycopg2 as the module for PostgreSQL? Thank you in advance for your help.

**Comment by foarsitter** [comment]
DRF and Python 3.11 is no problem, if you encounter problems with Django 4.2 you can try installing the master branch of DRF.

**Comment by auvipy** [comment, marked as answer]
django 4.2 is already tested in master branch in the CI and python 3.11 as well. you can try them from github directly in the meantime.
